packages/mui-material-nextjs/package.json

```diff
@@ -50,7 +50,7 @@
   "peerDependencies": {
     "@emotion/cache": "^11.11.0",
     "@emotion/server": "^11.11.0",
-    "@mui/material": "^5.0.0",
+    "@mui/material": "workspace:^",
```

Comment:
This is a peer dependency. Perhaps we need to add a dev dependency with `"workspace:^"`?

Reply:
Yes, I'd either keep it in peer dependencies and add a dev dependency to make the lockfile sound, or just remove it from peer dependencies altogether